integration/mediation-tests/tests-mediator-2/src/test/java/org/wso2/carbon/esb/message/processor/test/forwarding/StoreAndForwardWithEmptyMessageBodyTesCase.java

```diff
@@ -65,25 +65,25 @@ public void init() throws Exception {
                 "CREATE TABLE IF NOT EXISTS JDBC_MESSAGE_STORE(\n" + "indexId BIGINT(20) NOT NULL AUTO_INCREMENT,\n"
                         + "msg_id VARCHAR(200) NOT NULL ,\n" + "message BLOB NOT NULL,\n" + "PRIMARY KEY ( indexId )\n"
                         + ")");
-        logViewer = new LogViewerClient(context.getContextUrls().getBackEndUrl(), sessionCookie);
+        carbonLogReader = new CarbonLogReader();
+        carbonLogReader.start();
         super.init();
     }
 
     @Test
     public void testWithEmptyMessage() throws Exception {
-        logViewer.clearLogs();
+
         String location = getESBResourceLocation() + File.separator + "messageProcessorConfig" + File.separator
                 + "EmptyMsgBodyMessageStoreTest.xml";
         String proxyContent = FileUtils.readFileToString(new File(location));
         proxyContent = this.updateDatabaseInfo(proxyContent);
-        addMessageStore(AXIOMUtil.stringToOM(proxyContent));
-        loadESBConfigurationFromClasspath(
-                "artifacts" + File.separator + "ESB" + File.separator + "messageProcessorConfig" + File.separator
-                        + "EmptyMsgBodyTest.xml");
+
+        String storeLocation = System.getProperty("carbon.home")+"/repository/deployment/server/synapse-configs/default/message-stores/EmptyMsgBodyMessageStoreTest.xml";
```

Comment:
Shall we move this line to util method and pass  the 

folder name and artifact name as the argument to it ..

Reply:
we can pass **message-stores**  and **EmptyMsgBodyMessageStoreTest** this to it .. so that we can use it for other artifact types too.